packages/block-editor/src/components/link-control/index.js

```diff
@@ -220,21 +299,60 @@ function LinkControl( {
 		setIsEditingLink( false );
 	}
 
+	/**
+	 * Determines whether a given value could be a URL. Note this does not
+	 * guarantee the value is a URL only that it looks like it might be one
+	 * (thus the naming of the method).
+	 *
+	 * @param {string} val the candidate for being a valid URL (or not).
+	 */
+	function maybeURL( val ) {
```

Comment:
Assuming you already checked, but is there anything from `@wordpress/url` that can do this instead of needing to add another method?

Reply:
Good spot but `isURL` isn't concerned with `www.` or the internal format we need for Editor links. Therefore I've added these as custom checks. I don't think they'd get accepted in `@wordpress/url`. If they do then we can open another PR.